### ISSUE TITLE
CrossBrowser issue. Date range are not the same.

### ISSUE THREAD

**Issue by den-s** [wojtekmaj/react-calendar#160]
When I select range in Firefox range is:
0: Date 2018-12-28T22:00:00.000Z
1: Date 2018-12-30T21:59:59.999Z

When I select the same range in chrome, IE:
0: Sat Dec 29 2018 23:59:59 GMT+0200 (Eastern European Standard Time) {}
1: Sun Dec 30 2018 00:00:00 GMT+0200 (Eastern European Standard Time) {}

It is happens when the first day bigger than the second one.

**Comment by wojtekmaj** [comment]
Firefox and Chrome are just presenting the date differently. Chrome and IE shows you local time, Firefox shows UTC ("Z" at the end means UTC+0:00).

Firefox:
```
< new Date()
> Date 2018-12-27T10:28:45.876Z
```

Chrome, seconds after Firefox:
```
< new Date()
> Thu Dec 27 2018 11:28:48 GMT+0100 (czas środkowoeuropejski standardowy)
```

**Comment by den-s** [comment]
In FF - 2nd date 59:59
in chrome etc. - 1st date 59:59
http://prntscr.com/m03e8t
it is a bug )

**Comment by wojtekmaj** [comment]
No it is not. Read my message carefully. Print the dates using some methods that would give the same results, like `date.toISOString()` and compare them then to avoid confusion.

**Comment by wojtekmaj** [comment]
Ah, you mean date *order* is different, not the dates?

That was actually fixed in de19730bcbcf20703d5f72752465a8ced232f672 so should be released soon.
